CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# [4.18.0](https://github.com/algolia/instantsearch.js/compare/v4.16.1...v4.18.0) (2021-03-24)
+
+
+### Bug Fixes
+
+* **createURL:** correctly remove page in state ([#4679](https://github.com/algolia/instantsearch.js/issues/4679)) ([48c080e](https://github.com/algolia/instantsearch.js/commit/48c080ef85b974e68e1c80ceffea7a0138407a1e))
+* **utils:** circular dependency in createSendEventForHits ([#4680](https://github.com/algolia/instantsearch.js/issues/4680)) ([045f33b](https://github.com/algolia/instantsearch.js/commit/045f33bc6184fb04501e39a5a97e1e969095389a))
+
+
+### Features
+
+* **currentRefinements:** implement canRefine ([#4690](https://github.com/algolia/instantsearch.js/issues/4690)) ([f02416c](https://github.com/algolia/instantsearch.js/commit/f02416cf226ec3f7c2238b3e0902ec6f78381515))
```

Comment:
It would be nice to include all the `canRefine` APIs in the next version instead of introducing them one at a time.

Reply:
true, let's revert that one for now

Reply:
new pr: #4698